app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,262 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Api
+  extend ActiveSupport::Concern
+
+  included do
+    def one_sided_following
+      ExTwitter::Client.new.one_sided_following(self)
+    end
+
+    def one_sided_followers
+      ExTwitter::Client.new.one_sided_followers(self)
+    end
+
+    def mutual_friends
+      ExTwitter::Client.new.mutual_friends(self)
+    end
+
+    def common_friends(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_friends(self, other)
+    end
+
+    def common_followers(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_followers(self, other)
+    end
+
+    def removing
+      return [] if TwitterUser.where(uid: uid, user_id: user_id).limit(2).pluck(:id).size < 2
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        ExTwitter::Client.new.removing(old_one, new_one)
+      end.flatten
+    end
+
+    def removed
+      return [] if TwitterUser.where(uid: uid, user_id: user_id).limit(2).pluck(:id).size < 2
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        ExTwitter::Client.new.removed(old_one, new_one)
+      end.flatten
+    end
+
+    def replying(options = {})
+      begin
+        client.replying(__uid_i, options.merge(tweets: statuses)).map { |u| u.uid = u.id; u }
+      rescue => e
+        logger.warn "#{self.class}##{__method__} #{e.class} #{e.message}"
+        []
+      end
+    end
+
+    def replied(options = {})
+      if ego_surfing? && mentions.any?
+        result = mentions.map { |m| m.user }.map { |u| u.uid = u.id; u }
```

Comment:
__[rubocop]__
* Pass `&:user` as an argument to `map` instead of a block.
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)